changelog/@unreleased/pr-967.v2.yml

```
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Configuration cache
```

This is what goes in the releases page, let's make it clear what it's actually doing. https://github.com/palantir/gradle-git-version/releases/tag/3.3.0 is a good example, https://github.com/palantir/gradle-git-version/releases/tag/3.2.0 not so much.

```suggestion
  description: Make `gradle-git-version` configuration cache compatible.
```

---

Gotcha!